_integration-schemas/trello/users.md

```diff
@@ -0,0 +1,36 @@
+---
+tap: "trello"
+version: "1"
+key: ""
+
+name: "users"
+doc-link: "https://developer.atlassian.com/cloud/trello/rest/#api-boards-id-members-get"
+singer-schema: "https://github.com/singer-io/tap-trello/blob/master/tap_trello/schemas/users.json"
+description: |
+  The {{ table.name }} table contains information about users on your boards on your {{ integration.display_name }} account.
+
+replication-method: "Full Table"
+
+api-method:
+    name: "Get the Members of a Board"
+    doc-link: "https://developer.atlassian.com/cloud/trello/rest/#api-boards-id-members-get"
+
+attributes:
+  - name: "id"
+    type: "string"
+    primary-key: true
+    description: "The user ID."
+    #foreign-key-id: "users-id"
```

Comment:
I've marked a few other foreign keys for this in other tables - this should be uncommented once those foreign keys are added